### PR TITLE
International relocation payments reopen

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
International relocation payments reopen

### CODE REVIEW

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 1]
```diff
@@ -190,13 +188,13 @@ You will not be eligible for the international relocation payment if you are:
...
 
-You should apply for the IRP in the application window that opens at the start of your first term. If you miss this application window, you will have one further application window to apply – this will open at the start of your second term. If you do not apply within the first two application windows following the start of your job, you will no longer be eligible for the IRP.
+To remain eligible for the IRP, you must apply in either the first or second term of your employment as a teacher. Applications will reopen between May and June 2024.
 
 You will need a teaching job to apply for the IRP. [Learn more about finding a job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
-You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open in 2024.
+You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open later in 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:197]:
```suggestion
You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications reopen later in 2024.
```

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 2]
```diff
@@ -318,25 +316,25 @@ If you believe you meet the eligibility requirements, but have not been informed
...
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are open from 2 January 2024 to 29 February 2024. 
 
-You should apply for the IRP in the application window that opens at the start of your first term. If you miss this application window, you will have one further application window to apply – this will open at the start of your second term. If you do not apply within the first two application windows following the start of your course, you will no longer be eligible for the IRP.
+To remain eligible for the IRP, you must apply in either the first or second term of your teacher training course. Applications will reopen between May and June 2024.  
 
-You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open in 2024.
+If you have not yet started your salaried teacher training course, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open in 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:323]:
```suggestion
If you have not yet started your salaried teacher training course, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications reopen in 2024.
```

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 3]
```diff
@@ -190,13 +188,13 @@ You will not be eligible for the international relocation payment if you are:
 
 ## How teachers can apply for the international relocation payment
 
-Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk/) are currently closed. Teachers who missed the deadline for applications in 2023, or start their job from 1 January 2024, can apply when applications reopen between 2 January 2024 and 29 February 2024.
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for teachers are open from 2 January 2024 to 29 February 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:191]:
```suggestion
Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for teachers are open from 2 January 2024 to 29 February 2024.
```

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 4]
```diff
@@ -318,25 +316,25 @@ If you believe you meet the eligibility requirements, but have not been informed
 
 ### Salaried trainees
 
-Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk/) are currently closed. Salaried trainees who missed the deadline for applications in 2023, or start their course from 1 January 2024, can apply when applications reopen between 2 January 2024 and 29 February 2024.
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are open from 2 January 2024 to 29 February 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:319]:
```suggestion
Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are open from 2 January 2024 to 29 February 2024.
```

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 5]
```diff
@@ -318,25 +316,25 @@ If you believe you meet the eligibility requirements, but have not been informed
 
 ### Salaried trainees
 
-Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk/) are currently closed. Salaried trainees who missed the deadline for applications in 2023, or start their course from 1 January 2024, can apply when applications reopen between 2 January 2024 and 29 February 2024.
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are open from 2 January 2024 to 29 February 2024. 
 
-You should apply for the IRP in the application window that opens at the start of your first term. If you miss this application window, you will have one further application window to apply – this will open at the start of your second term. If you do not apply within the first two application windows following the start of your course, you will no longer be eligible for the IRP.
+To remain eligible for the IRP, you must apply in either the first or second term of your teacher training course. Applications will reopen between May and June 2024.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:321]:
```suggestion
To remain eligible for the IRP, you must apply in either the first or second term of your teacher training course. Applications will reopen between May and June 2024.
```

**Review by gemmadallmandfe — COMMENTED** [review]
@laurahermionetennant92 I've left some minor comments on this PR given that Maxine will have very little time to review on 2 January before it needs merging

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md (hunk 3 above)
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:191]: I might be going mad @gemmadallmandfe but I can't see what's changed here?

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md (hunk 4 above)
Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:319]: Or here?

**Review by MaxineCl — APPROVED** [review]

**Pull request merged**